### PR TITLE
[BEAM-584] Support Verifiers in TestSparkRunner

### PR DESCRIPTION
**Pull request by aviemzur**

### CODE REVIEW

**Comment by aviemzur** [comment]
@amitsela @jasonkuster

File: examples/java/pom.xml [hunk 1]
```diff
@@ -97,6 +97,12 @@
           <version>${spark.version}</version>
           <scope>runtime</scope>
           <optional>true</optional>
+          <exclusions>
```
Comment (amitsela) [inline examples/java/pom.xml]: Already handled by #892

**Comment by amitsela** [comment]
@aviemzur please update your branch from master.

**Comment by amitsela** [comment]
LGTM. 
Ping @jasonkuster ?

**Comment by jasonkuster** [comment]
LGTM. Thanks for taking this on @aviemzur!

**Pull request closed without merging**